packages/orn-locator/src/resolvers/books.ts

```diff
@@ -0,0 +1,155 @@
+import fetch from 'node-fetch';
+
+const oswebUrl = 'https://openstax.org/apps/cms/api/v2/pages';
+const fields = 'cnx_id,authors,publish_date,cover_color,amazon_link,book_state,promote_image,webview_rex_link,cover_url,title_image_url';
+
+const getArchiveInfo = async (bookId: string) => {
```

Comment:
i would prefer to not do all these extra queries to load this data, it would be nice if maybe REX published json files with convenient names, since the rex urls are a thing that we need and i'm re-producing the rex routing logic for the urls here (not that it has every changed)